src/routes/RequestsRoute.js

```diff
@@ -1244,6 +1373,28 @@ class RequestsRoute extends React.Component {
                     values={{ sp: servicePointName }}
                   />
                 </PrintButton>
+                <PrintButton
+                  buttonStyle="dropdownItem"
+                  id="printPickSlipsBtn1"
+                  disabled={isPickSlipsEmpty || selectedRowsNonPrintable(pickSlipsData, selectedRows)}
```

Comment:
Could you create separate constant for this condition `isPickSlipsEmpty || selectedRowsNonPrintable(pickSlipsData, selectedRows)`?